src/sad_spirit/pg_builder/Parser.php

```diff
@@ -22,6 +22,8 @@
 
 use Psr\Cache\CacheItemPoolInterface;
 use Psr\Cache\InvalidArgumentException;
+use sad_spirit\pg_builder\exceptions\SyntaxException;
+use sad_spirit\pg_builder\nodes\lists\TargetList;
```

Comment:
Please don't `use` the package classes, use relative namespaces as everywhere else